src/Components/PresentationalComponents/Filters/CustomFilters/CheckboxCustomFilter.js

```diff
@@ -48,10 +48,11 @@ const CheckboxCustomFilter = ({
             isOpen={isOpen}
             placeholderText={getPlaceholderText()}
             key={filterId}
-            width="auto"
+            width={selectProps?.width || 'auto'}
             onSelect={(event, optionName) => { handleOnCheckboxChange(filterId, optionName); }}
             selections={filterData[filterId]}
             isCheckboxSelectionBadgeHidden
+            direction={selectProps?.direction || 'down'}
```

Comment:
Since there is already `{...selectProps}` at the end there is no need to use `direction={selectProps?.direction || 'down'}` or `width={selectProps?.width || 'auto'}` respectively. Because `{...selectProps}` will make sure that these props are replaced with the values from the `selectProps` object.

Reply:
If I don't add those, then the Report by CVEs modal, which also uses these filters, doesn't set width to 'auto'. So I'd have to manually send that up each place where CheckboxCustomFilter is called. We can remove the `direction` though.

Reply:
If `selectProps` object contains property `width`, this line
`width={selectProps?.width || 'auto'}`
will be overwritten by
`{...selectProps}`

Thus
`width={selectProps?.width || 'auto'}`
can be simplified to
`width="auto"`